string scenario
---
string orig_scenario
string novel_scenario 
int64 T